styleguide/source/assets/scss/02-molecules/_press-release-logos.scss

```diff
@@ -0,0 +1,12 @@
+.press-release-logos {
```

Comment:
This should be set to `display:none;` since by default and the rest of the display information should be wrapped in a breakpoint since the logo group does not display on the mobile wireframe.